rpm/gstreamer.spec

```diff
@@ -129,20 +129,18 @@ rm -fr $RPM_BUILD_ROOT/%{_datadir}/gdb
 %{_libdir}/libgstnet-%{majorminor}.so.*
 %{_libdir}/gstreamer-%{majorminor}/libgstcoreelements.so
 %{_libdir}/gstreamer-%{majorminor}/libgstcoretracers.so
+%{_libexecdir}/gstreamer-%{majorminor}/gst-completion-helper
 %{_libexecdir}/gstreamer-%{majorminor}/gst-plugin-scanner
-%{_libexecdir}/gstreamer-%{majorminor}/gst-ptp-helper
```

Comment:
Why is this getting disabled? If there's a reason, could also note that in the commit message.

Reply:
I had disabled that because it needed rust but figured out how to make it build by adding rust dependency and needed environmental variables.